x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/min_age_field/min_age_field.tsx

```diff
@@ -62,13 +63,25 @@ const i18nTexts = {
       defaultMessage: 'nanoseconds',
     }
   ),
+  rolloverToolTipDescription: i18n.translate(
+    'xpack.indexLifecycleMgmt.editPolicy.minimumAge.rolloverToolTipDescription',
+    {
+      defaultMessage:
+        'Data age is calculated from rollover. Rollover is configured in the hot phase.',
```

Comment:
I'd avoid using "data" generally. It could mean "document" or "index." 
```suggestion
        'Calculated as time since rollover. Rollover is configured in the hot phase.',
```

I'm also not 100% sure that this tooltip is clearer than the original copy. `<N> days from rollover` seems clearer.